app/scripts/lib/setupSentry.js

```diff
@@ -146,31 +154,88 @@ export default function setupSentry({ release, getState }) {
       } else {
         return null;
       }
-      return breadcrumb;
+      return rewriteBreadcrumb(breadcrumb);
     },
   });
 
-  function rewriteReport(report) {
-    try {
-      // simplify certain complex error messages (e.g. Ethjs)
-      simplifyErrorMessages(report);
-      // modify report urls
-      rewriteReportUrls(report);
-      // append app state
-      if (getState) {
-        const appState = getState();
-        if (!report.extra) {
-          report.extra = {};
-        }
-        report.extra.appState = appState;
+  return Sentry;
+}
+
+function hideUrlIfNotInternal(_url) {
```

Comment:
Conventionally, a leading underscore implies that a parameter is unused. This one is being used though.

Reply:
Removed in latest commit https://github.com/MetaMask/metamask-extension/pull/16780/commits/daaac950566477c5197089c509babada82f9cdaa#diff-a8c55d1ad0724e946c8c35114d6a298b74904accdddedff77e1b8ade2097fd65L164